tonic-examples/Cargo.toml

```diff
@@ -80,7 +80,7 @@ tower = { git = "https://github.com/tower-rs/tower" }
 # Required for routeguide
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
-rand = "0.7.2"
+rand = "0.6"
```

Comment:
why rand 0.6?

Reply:
Sorry, didn't see this. Because at the time I wrote this, some other dependency (some tokio sub-crate, I think) wall pulling in 0.6, so I changed it to match.

Reply:
I guess once 0.1 is close to finished we can sweep through all deps and remove what is not essential.